packaging/ansible-runner-service-project/project/roles/ovirt-ova-pack/tasks/main.yml

```diff
@@ -1,16 +1,34 @@
 ---
-- name: Run packing script
-  script: >
-    pack_ova.py
-    "{{ entity_type }}"
-    "{{ ova_file.dest }}"
-    "{{ ovirt_ova_pack_ovf }}"
-    "{{ ovirt_ova_pack_disks }}"
-    "{{ ovirt_ova_pack_tpm }}"
-    "{{ ovirt_ova_pack_nvram }}"
-    "{{ ovirt_ova_pack_padding }}"
-  args:
-    executable: "{{ ansible_python_interpreter }}"
-  register: packing_result
-  ignore_errors: yes
-  when: ova_file is defined and ova_file.dest is defined
+- block:
+  - name: Create temporary directory
+    tempfile:
+      state: directory
+      suffix: ova
+    register: ova_temp
+
+  - name: Copy pack_ova.py to temp directory
+    copy:
+      src: pack_ova.py
+      dest: "{{ ova_temp.path }}/pack_ova.py"
```

Comment:
why we need the temporary directory all the sudden on these flows?

Reply:
Explained in the comments earlier, but in short `script` does not support `async`. In order to run `command` we need to copy those scripts to host.

Reply:
Right right, thanks. I missed it.